client/js/lounge.js

```diff
@@ -1331,7 +1331,7 @@ $(function() {
 					} else {
 						var current = $(this);
 
-						if (current.attr("class") === "date-marker") {
+						if (current.hasClass("date-marker") && prev.hasClass("date-marker")) {
```

Comment:
That's correct. And you might be right the code is flawed. Any ideas on how to improve it?